ngsild-payloads/applicationPod.jsonld

```diff
@@ -0,0 +1,89 @@
+{
+  "id": "urn:ngsi-ld:ApplicationPod:istio-ingressgateway-9c8b9b586-thl7x",
+  "type": "ApplicationPod",
+  "name": {
+    "type": "Property",
+    "value": "istio-ingressgateway-9c8b9b586-thl7x"
+  },
+  "temporalResolution": {
+    "type": "Property",
+    "value": "PT1H"
+  },
+  "cpuUtilization": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 0.06907686666721323,
+      "unitCode": "CORE"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 0.07907686666721322,
+      "unitCode": "CORE"
+    }
+  ],
+  "networkTransmission": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 196.1,
+      "unitCode": "B"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 200.1,
+      "unitCode": "B"
+    }
+  ],
+  "networkReception": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 177.46666666666667,
+      "unitCode": "B"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 181.46666666666667,
+      "unitCode": "B"
+    }
+  ],
+  "ramUtilization": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 92.703125,
+      "unitCode": "MB"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 96.703125,
+      "unitCode": "MB"
+    }
+  ],
+  "diskIo": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 14882.1333,
+      "unitCode": "KB"
+    },
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:05:00Z",
+      "value": 13774.2767,
+      "unitCode": "KB"
+    }
+  ],
+  "deployedInNode": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:Node:cube04"
```

Comment:
you seem to introduce a new type of entity (`Node`), could you also add a sample (and declare its type and attributes in the context)?